packages/documentation/plugins/mdx-pages/gatsby-node.js

```diff
@@ -66,6 +67,9 @@ exports.createPages = async ({ graphql, actions }) => {
         context: {
           id: node.id,
           source: 'component',
+          frontmatter: {
+            ...node.frontmatter,
+          },
           name: node.frontmatter.name,
```

Comment:
Turns out I don't need to query the Mdx nodes- I needed to expand the front matter query and then add it to the context. I'll probably need to expand this to include tags but this fixes the hardcoded path issue.